DEV/org.openl.core.annotations/src/org/openl/binding/impl/method/NonNullLiteral.java

```diff
@@ -0,0 +1,11 @@
+package org.openl.binding.impl.method;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(value = { ElementType.PARAMETER })
+public @interface NonNullLiteral {

```

Comment:
Add JavaDocs description of the purpose of this annotation, an example of usage.